### DISCUSSION TITLE
스터디 관련 규칙

### DISCUSSION THREAD

**Discussion by fkdl0048** [BRIDGE-DEV/Unity_Custom_Editor_Study#1] [Announcements]
전체적인 스케줄이나 진행사항은 이 레포에서 진행할 것 같습니다.  

공개적인 학습은 항상 긍정적인 효과를 내기 때문에 Public으로 진행.. 따라서 스터디인원 이외의 의견도 적극 수용합니다!  

## 주 1회 온라인 회의 금요일 오후 8시에 진행  

회의 때 진행했으면 하는 내용

1. 주 단위 목표 설정
전 주에 설정했으나 구현하지 못한 문제나 예외 발생 시 동적인 목표 설정이 필요  

2. 주 단위 회고  
진행하면서 느낀점, 개선방향등 편하게 이야기 하기

## 작업은 투명하게 

1. 기능의 리스트를 뽑아서 issue로 관리 진행 정도는 project를 활용
지속적인 논의가 필요하지만 기능들의 리스트를 뽑고 우선순위로 project단에서 관리

2. 풀리퀘를 통해서 작업을 검증
리뷰어로 등록해서 리뷰후 merge를 진행

개인적으로 코드의 평가, 조언은 작업중인 프로그램의 품질을 올리기 위함이지 사람을 평가하는게 아니라고 생각합니다.  
고쳐야 할 부분이 있으면 편하게 말씀해주시면 감사하겠습니다!

**Comment by fkdl0048** [comment]
추가되었으면 하는 규칙이나 이상한 규칙들은 말씀해주시면 수정하겠습니다..!

**Reply by kugorang** [reply]
더 많은 사람들에게 우리가 공부한 내용이 알려졌으면 좋겠습니다.

1. 브릿지의 경우, 깃허브를 안 보시는 분들이 많으므로 격주 간격으로 카톡방에 공유하면 좋겠습니다.
2. 유니티와 관련된 외부 커뮤니티에도 저희 공부 내용을 공유했으면 좋겠습니다.
> 유니티 개발자 커뮤니티 (페이스북 그룹) - https://www.facebook.com/groups/unitykorea/
> 유니티 허브 (네이버 카페) - https://cafe.naver.com/unityhub
